File: 0881 boats-to-save-people/solution.cs
public class Solution {
    public int NumRescueBoats(int[] people, int limit) {
        Array.Sort(people);
        int i=0,j=people.Length-1,c=0;
        while(i<=j) {
            if(people[i]+people[j] <= limit ) i++;
            j--;
            c++;
        }
        return c;
    }
}

File: 0881 boats-to-save-people/solution.java
class Solution {
    public int numRescueBoats(int[] people, int limit) {
        int c=0,i=0,j=people.length-1;
        Arrays.sort(people);
        while(i<=j) {
            int sum=people[i]+people[j];
            if(sum<=limit) {
               i++;
            }
            j--;
            c++;
        }
        return c;
    }
}
